[2024-04-29T05:38:21.324+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl.get_files scheduled__2024-04-20T00:00:00+00:00 [queued]>
[2024-04-29T05:38:21.362+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl.get_files scheduled__2024-04-20T00:00:00+00:00 [queued]>
[2024-04-29T05:38:21.362+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-04-29T05:38:21.399+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): get_files> on 2024-04-20 00:00:00+00:00
[2024-04-29T05:38:21.438+0000] {standard_task_runner.py:60} INFO - Started process 2241 to run task
[2024-04-29T05:38:21.436+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl', 'get_files', 'scheduled__2024-04-20T00:00:00+00:00', '--job-id', '38', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpe36u_72r']
[2024-04-29T05:38:21.497+0000] {standard_task_runner.py:88} INFO - Job 38: Subtask get_files
[2024-04-29T05:38:21.820+0000] {task_command.py:423} INFO - Running <TaskInstance: etl.get_files scheduled__2024-04-20T00:00:00+00:00 [running]> on host 422c09390d6c
[2024-04-29T05:38:22.582+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl' AIRFLOW_CTX_TASK_ID='get_files' AIRFLOW_CTX_EXECUTION_DATE='2024-04-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-20T00:00:00+00:00'
[2024-04-29T05:38:22.587+0000] {etl.py:24} INFO - 5 files found in /opt/***/dags/data
[2024-04-29T05:38:22.588+0000] {python.py:202} INFO - Done. Returned value was: ['/opt/***/dags/data/github_events_02.json', '/opt/***/dags/data/github_events_03.json', '/opt/***/dags/data/github_events_04.json', '/opt/***/dags/data/github_events_05.json', '/opt/***/dags/data/github_events_01.json']
[2024-04-29T05:38:22.674+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=etl, task_id=get_files, execution_date=20240420T000000, start_date=20240429T053821, end_date=20240429T053822
[2024-04-29T05:38:22.869+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-29T05:38:23.127+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
